fabric_plugin/tunnel.py

```diff
@@ -42,22 +39,27 @@ def accept(channel, (src_addr, src_port), (dest_addr, dest_port)):
         try:
             sock.connect((local_host, local_port))
         except Exception as e:
+            try:
+                channel.close()
+            except Exception as e2:
+                close_error = ' (While trying to close channel: {0})'.format(
+                    e2)
+            else:
+                close_error = ''
```

Comment:
`else` is only evaluated if no exception occurred, in which case, we want an empty string. This is actually the expected state because we assume close will usually work.